atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -110,6 +111,7 @@ class OrchestrateBuildPlugin(BuildStepPlugin):
     """
 
     EXCLUDE_FILENAME = 'exclude-platform'
+    UNREACHABLE_CLUSTER_LOAD = -1
```

Comment:
It definitely doesn't belong in `get_enabled_clusters_for_platform`, which should only be concerned with the content of the configuration file.